### PR TITLE
[mg]: Add inspec tests

### PR DESCRIPTION
**Pull request by gavindidrichsen**
```rspec
hab pkg exec chef/inspec inspec exec /src/. --chef-license=accept -t docker://63e2f2280f7331ef7f0bbef111495f71533d881a753305be0494c94d561ce65f --input-file /src/attributes.yml

Profile: Habitat Core Plan mg (mg)
Version: 0.1.0
Target:  docker://63e2f2280f7331ef7f0bbef111495f71533d881a753305be0494c94d561ce65f

  ✔  core-plans-mg-exists: Ensure mg exists
     ✔  Command: `hab pkg path core/mg` exit_status is expected to eq 0
     ✔  Command: `hab pkg path core/mg` stdout is expected not to be empty
     ✔  Command: `hab pkg path core/mg` stderr is expected to be empty
     ✔  File /hab/pkgs/core/mg/20180408/20200615162002/bin/mg is expected to exist
  ✔  core-plans-mg-works: Ensure mg works as expected
     ✔  Command: `hab pkg path core/mg` exit_status is expected to eq 0
     ✔  Command: `hab pkg path core/mg` stdout is expected not to be empty
     ✔  Command: `hab pkg path core/mg` stderr is expected to be empty
     ✔  Command: `/hab/pkgs/core/mg/20180408/20200615162002/bin/mg --version` exit_status is expected not to eq 0
     ✔  Command: `/hab/pkgs/core/mg/20180408/20200615162002/bin/mg --version` stderr is expected not to be empty
     ✔  Command: `/hab/pkgs/core/mg/20180408/20200615162002/bin/mg --version` stderr is expected to match /usage: mg/
     ✔  Command: `/hab/pkgs/core/mg/20180408/20200615162002/bin/mg --version` stdout is expected to be empty


Profile Summary: 2 successful controls, 0 control failures, 0 controls skipped
Test Summary: 11 successful, 0 failures, 0 skipped
+ set +x
[28][default:/src:0]# 
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,64 @@
...
+
+You can now use the binary as normal.  For example:
+
+``/bin/mg --help`` or ``mg --help``
+
+```bash
+$ mg --help
+mg: invalid option -- '-'
+usage: mg [-nR] [-f mode] [+number] [file ...]
+```
```
Comment (MindNumbing) [inline README.md]: Might be best to show an example of a working command here.

**Review by MindNumbing — CHANGES_REQUESTED** [review]
One thing to look at, otherwise good.

File: README.md (hunk 1 above)
Comment (gavindidrichsen) [reply README.md]: Hi @MindNumbing.  As this is a emacs-like application, I've changed command example to show a picture instead.

**Review by MindNumbing — APPROVED** [review]
Thanks @gavindidrichsen, LGTM! 👍

**Pull request merged**